### DISCUSSION TITLE
can't run valid clusterbomb template

### DISCUSSION THREAD

**Discussion by marz-hunter** [projectdiscovery/nuclei#2311] [Q&A] [answered]
when validating the template it is valid but when run it shows error Could not run nuclei: no valid templates were found

template: https://github.com/projectdiscovery/nuclei-templates/blob/123939999ea71f136822b3607cd3a441c07d8b44/fuzzing/header-command-injection.yaml

![nuclei](https://user-images.githubusercontent.com/58464282/179655980-b5ad64e2-26c1-44cf-8f55-fb083da79aa9.PNG)

**Comment by marz-hunter** [comment]
nuclei version Current Version: 2.7.4 os ubuntu 18

**Comment by ehsandeep** [comment, marked as answer]
```
nuclei -t fuzzing/header-command-injection.yaml -itags fuzz
```

@marz-hunter, you need to use `-itags` to include tags that are excluded as default.

**Reply by marz-hunter** [reply]
thanks
